README.md

```diff
@@ -48,20 +44,36 @@ by running it with the `--help` flag or reading the [rustc dev guide][rustcguide
 [gettingstarted]: https://rustc-dev-guide.rust-lang.org/getting-started.html
 [rustcguidebuild]: https://rustc-dev-guide.rust-lang.org/building/how-to-build-and-run.html
 
-### Building on a Unix-like system
-1. Make sure you have installed the dependencies:
-
-   * `g++` 5.1 or later or `clang++` 3.5 or later
-   * `python` 3 or 2.7
-   * GNU `make` 3.81 or later
-   * `cmake` 3.13.4 or later
-   * `ninja`
-   * `curl`
+### Dependencies
+
+Make sure you have installed the dependencies:
+
+   * `python` 3 or 2.7 (under the name python; python2 or python3 will not work)
    * `git`
-   * `ssl` which comes in `libssl-dev` or `openssl-devel`
+   * OpenSSL (`libssl-dev` or `openssl-devel` on most Unix distros)
+   * XZ (`liblzma-dev` on most distros)
+   * `libffi-dev`
+   * `curl` (not needed on Windows)
    * `pkg-config` if you are compiling on Linux and targeting Linux
+   * `libiconv` (already included with glibc on Debian-based distros)
```

Comment:
XZ, libffi, and libiconv are new in this list (not from the dev-guide). I found them by looking at the nix build recipe: https://github.com/NixOS/nixpkgs/blob/5528350186a9e826588cee1329640899ca44a0cf/pkgs/development/compilers/rust/rustc.nix#L1-L16
`iconv` is bundled with glibc on most distros, but miri is definitely using libffi and llvm and `rust-installer` are using XZ.